pkg/util/condition_test.go

```diff
@@ -0,0 +1,54 @@
+package util
+
+import (
+	"testing"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	workv1alpha2 "github.com/karmada-io/karmada/pkg/apis/work/v1alpha2"
+)
+
+const (
+	scheduleSuccessReason  = "BindingScheduled"
+	scheduleFailedReason   = "BindingFailedScheduling"
+	scheduleSuccessMessage = "Binding has been scheduled"
+	scheduleFailedMessage  = "Binding failed scheduling"
```

Comment:
Since we are testing the `IsConditionsEqual` function, we don't care if this is a real event or not. So, I suggest using a fake event, like 
```suggestion
	fakeEventSuccessReason  = "Success"
	fakeEventFailedReason   = "Failed"
	fakeSuccessMessage = "Successfull event message"
	fakeFailedMessage  = "Failed event message"
```